### ISSUE TITLE
twisted.protocols.amp.Float.toString returns a str, not bytes on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8746]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8746|
|Type|defect|
|Created|2016-08-11 23:17:33Z|
|Branch|https://github.com/twisted/twisted/tree/8746-amp-float|

<details><summary>Searchable metadata</summary>

```
trac-id__8746 8746
type__defect defect
reporter__hawkowl hawkowl
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8746_amp_float 8746-amp-float
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1470957453421040 1470957453421040
changetime__1471799472961662 1471799472961662
version__None None
owner__Amber_Brown__hawkowl_____ Amber Brown <hawkowl@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

PR: https://github.com/twisted/twisted/pull/486

**Comment by twisted-trac** [comment]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)<a name="note_3"></a>|@markrwilliams commented|
|-|-|

Nice and simple!  I went ahead and merged in master.  Once the builds are green, this is ready to merge; I'll do so if I see everything pass before anybody else does.

Thanks!!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set owner to @hawkowl<br>@hawkowl set status to `closed`|
|-|-|

In changeset 1d408126062f6cb8f214c97b8183dd612f86d177
```
#!CommitTicketReference repository="" revision="1d408126062f6cb8f214c97b8183dd612f86d177"
Merge 8746-amp-float: Fix Floats in AMP on Python 3

Author: hawkowl
Reviewer: markrwilliams
Fixes: #8746
```
